src/Elements/Comment.react.js

```diff
@@ -38,16 +57,111 @@ class Comment extends Component {
                             <span className="comment-name">{name}</span>
                             <span className="comment-time">{this._renderTime()}</span>
                         </div>
+                        { isEditable && (canEdit || canRemove) ?
+                            isEditMode ? (
+                                <div
+                                    style={{
+                                        flex: '1 0 auto',
+                                        margin: detailsPosition === 'right' ? '0 0 0 11px' : '0 11px 0 0',
+                                        textAlign: detailsPosition === 'right' ? 'left' : 'right'
+                                    }}
+                                >
+                                    <a
+                                        className="cancel-link"
+                                        onClick={this._onCancelEditClick}
+                                        style={{ marginRight: '11px' }}
+                                    >
+                                        {'Cancel'}
+                                    </a>
+                                    <a
+                                        className="save-link"
+                                        disabled={!canEdit}
+                                        onClick={this._onSaveClick}
+                                        style={{ marginRight: '11px' }}
+                                    >
+                                        {'Save'}
+                                    </a>
+                                    <a
+                                        className="remove-link"
```

Comment:
I added classes like these so consumers can apply styles.  For instance, in the comps (https://sketch.cloud/s/ORzY2/Pr3OEv) the "Remove" CTA is black versus "Edit" and "Save" being our standard link color.  I didn't want to bake that in ... I was thinking rather that consumer can apply CSS to do these kinds of customizations.  Is this a good approach?  Anything else in here need CSS classes?  Do you think we need props anywhere to be able to apply additional classes (like we do elsewhere in the React CM UI library)?